runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1601,6 +1770,95 @@ void J9::ARM64::PrivateLinkage::buildVirtualDispatch(TR::Node *callNode,
+   // PIC call inline and a virtual call or dynamic PIC call out of line.
+   //
+   // Otherwise generate a reasonable amount of static PIC calls and a
+   // virtual call or dynamic PIC call all inline.
+   //
+   if (callIsSafe && !callNode->isTheVirtualCallNodeForAGuardedInlinedCall() && !comp()->getOption(TR_DisableInterpreterProfiling))
+      {
+      static uint32_t  maxVirtualStaticPICs = comp()->getOptions()->getMaxStaticPICSlots(comp()->getMethodHotness());
+      static uint32_t  maxInterfaceStaticPICs = comp()->getOptions()->getNumInterfaceCallCacheSlots();
+
+      TR_ScratchList<J9::ARM64PICItem> values(cg()->trMemory());
+      const uint32_t maxStaticPICs = methodSymbol->isInterface() ? maxInterfaceStaticPICs : maxVirtualStaticPICs;
+
+      if (getProfiledCallSiteInfo(cg(), callNode, maxStaticPICs, values))
+         {
+         ListIterator<J9::ARM64PICItem>  i(&values);
+         J9::ARM64PICItem               *pic = i.getFirst();
+
+         // If this value is dominant, optimize exclusively for it
+         if (pic->_frequency > MAX_PROFILED_CALL_FREQUENCY)
+            {
+            if (comp()->getOption(TR_TraceCG))
+               {
+               traceMsg(comp(), "Found dominant profiled target, frequency = %f\n", pic->_frequency);
+               }
+            TR::LabelSymbol *slowCallLabel = generateLabelSymbol(cg());
+
+            TR::Instruction *gcPoint = buildStaticPICCall(cg(), callNode, pic->_clazz, pic->_method,
+                                                            vftReg, x9, slowCallLabel, regMapForGC);
+            generateLabelInstruction(cg(), TR::InstOpCode::label, callNode, doneLabel, dependencies);
+
+            // Out of line virtual/interface call
+            //
+            TR_ARM64OutOfLineCodeSection *slowCallOOL = new (trHeapMemory()) TR_ARM64OutOfLineCodeSection(slowCallLabel, doneLabel, cg());
+
+            slowCallOOL->swapInstructionListsWithCompilation();
+            TR::Instruction *OOLLabelInstr = generateLabelInstruction(cg(), TR::InstOpCode::label, callNode, slowCallLabel);
+
+            // XXX: Temporary fix, OOL instruction stream does not pick up live locals or monitors correctly.
+            TR_ASSERT_FATAL(!OOLLabelInstr->getLiveLocals() && !OOLLabelInstr->getLiveMonitors(), "Expecting first OOL instruction to not have live locals/monitors info");
+            OOLLabelInstr->setLiveLocals(gcPoint->getLiveLocals());
+            OOLLabelInstr->setLiveMonitors(gcPoint->getLiveMonitors());
+
+            TR::LabelSymbol *doneOOLLabel = generateLabelSymbol(cg());
+
+            if (methodSymbol->isInterface())
+               {
+               TR::LabelSymbol              *ifcSnippetLabel = generateLabelSymbol(cg());
+               TR::ARM64InterfaceCallSnippet *ifcSnippet =     new (trHeapMemory()) TR::ARM64InterfaceCallSnippet(cg(), callNode, ifcSnippetLabel, argSize, doneOOLLabel, (uint8_t *)thunk);
```

Comment:
Please align the column position of the variables.

Reply:
I fixed it.